build/configs/esp32_DevKit/README.md

```diff
@@ -0,0 +1,110 @@
+# esp32_DevKit
+
+Espressif Development Kit for ESP32 chipset.
+
+## Contents
+
+> [Information](#information)  
+> [Environment Set-up](#environment-set-up)  
+ 
+## Information
+
+will be updated
+
+## Environment Set-up
```

Comment:
change the title to  'Manual Environment Set-up', and leave a comment like 'docker method will be updated later '

Reply:
fixed, thanks.